profitbricks-rest/src/main/java/org/apache/jclouds/profitbricks/rest/binder/server/AttachCdromRequestBinder.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.jclouds.profitbricks.rest.binder.server;
+
+import com.google.gson.Gson;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.jclouds.profitbricks.rest.binder.BaseProfitBricksRequestBinder;
+import org.apache.jclouds.profitbricks.rest.domain.Server;
+import org.jclouds.http.HttpRequest;
+
+public class AttachCdromRequestBinder extends BaseProfitBricksRequestBinder<Server.Request.AttachCdromPayload> {
+
+   protected final Map<String, Object> requestBuilder;
+   String dataCenterId;
+   String serverId;
+
+   AttachCdromRequestBinder() {
+      super("cdrom");
+      this.requestBuilder = new HashMap<String, Object>();
+   }
+
+   @Override
+   protected String createPayload(Server.Request.AttachCdromPayload payload) {
+
+      dataCenterId = payload.dataCenterId();
+      serverId = payload.serverId();
+      
+      requestBuilder.put("id", payload.imageId());
+      
+      return (new Gson()).toJson(requestBuilder);
```

Comment:
As @devcsrj commented elsewhere, can you inject the Json type instead and use it? We shouldn't couple binders to a concrete json library.